src/nools/lib.js

```diff
@@ -1,5 +1,34 @@
+function bindAllFunctionsToContext(obj, context) {
+  var keys = Object.keys(obj);
+  for (var i in keys) {
+    var key = keys[i];
+    switch(typeof obj[key]) {
+      case 'object':
+        bindAllFunctionsToContext(obj[key], context);
+        break;
+      case 'function':
+        obj[key] = obj[key].bind(context);
```

Comment:
This means that nobody should ever use arrow functions when configuring. We should make sure people know this?

Reply:
You can use lambdas and they will be totally supported without issue, just the `this` keyword won't work. It isn't required for most functions. But yes -- it should certainly be documented. I'll make a note of that in the doc changes I'm preparing.